compiler/tests/generated/alloc_ctors/attributes_types.cpp

```diff
@@ -10,9 +10,10 @@ namespace tests
     namespace Enum
     {
         const
-        std::map<std::string, enum Enum> _name_to_value_Enum =
-            boost::assign::map_list_of<std::string, enum Enum>
-                ("Value1", Value1);
+        std::map<std::string, enum Enum> _name_to_value_Enum
+            {
+                { "Value1", Value1 }
```

Comment:
Inserting into a map is going to be a bit faster if the keys are already in order. Can we sort them in Haskell?